src/libraries/System.Text.Json/gen/Resources/xlf/Strings.cs.xlf

```diff
@@ -142,16 +142,6 @@
         <target state="translated">Typ obsahuje více členů s komentářem JsonExtensionDataAttribute</target>
         <note />
       </trans-unit>
-      <trans-unit id="TypeContainsRefLikeMemberFormat">
```

Comment:
This is undoing a change made in August https://github.com/dotnet/runtime/commit/c1ab6c5b2524880ed9368841a0a5d8ead78ea09d#diff-41f0a49909511e29afd9f8ee632594843d510f106249d7cede193017500dba57R145